R/tostring.R

```diff
@@ -59,22 +300,25 @@ do_cell_fnotes_wrap <- function(mat, widths, max_width, tf_wrap) {
     unlist(mapply(wrap_string,
       str = mfs,
       width = cell_widths_mat,
-      collapse = "\n"
+      collapse = "\n",
+      MoreArgs = list(fontspec = fontspec)
     )),
     ncol = ncol(mfs)
   )
 
-  ## XXXXX this is wrong and will break for listings cause we don't know when
-  ## we need has_topleft to be FALSE!!!!!!!!!!
-  mat <- mform_handle_newlines(mat)
-
-  ## this updates extents in rinfo AND nlines in ref_fnotes_df
-  mat <- update_mf_nlines(mat, max_width = max_width)
+  if (expand_newlines) {
```

Comment:
is there a case where we do not want it?

Reply:
Yes, I use it in the jj soon-to-be opensourced rtf exporting function. can go over that more on wed if desired, though its not inside this pr.